lib/runner.js

```diff
@@ -55,6 +57,10 @@ optionChain(chainableMethods, function (opts, title, fn) {
 		opts.serial = true;
 	}
 
+	if (this._match) {
+		opts.skipped = matcher([title], this._match).length === 0;
```

Comment:
This function is also called when adding hooks, which themselves can be titled. Would be weird to skip those. I think you should be able to guard against that by checking `opts.type === 'test'`.